rest/model/test.go

```diff
@@ -101,7 +101,8 @@ func (at *APITest) BuildFromService(st interface{}) error {
 		at.StartTime = utility.ToTimePtr(v.Start)
 		at.EndTime = utility.ToTimePtr(v.End)
 		duration := v.End.Sub(v.Start)
-		at.Duration = float64(duration)
+		// Convert from nanosecond to millisecond
+		at.Duration = float64(duration) / float64(time.Second)
```

Comment:
I think the time library has a built in conversion, something like
```
at.Duration = float64(duration.Milliseconds())
```

Reply:
Thanks